onnxconverter_common/onnx2py.py

```diff
@@ -211,8 +235,9 @@ def convert(model, out_path):
     const_counter = 0
     TracingObject.reset_cnt(clear_field_traced)
     TracingObject.reset_cnt(make_external_tensor_traced)
+    DATA_DIR_TRACED = TracingObject("DATA_DIR", const_dir)
 
-    model_trace = convert_model(model)
+    model_trace = convert_field(model)
     code = "from onnx import helper, numpy_helper, TensorProto"
     if TracingObject.get_cnt(make_external_tensor_traced):
```

Comment:
Looks great! Suggest to add one more comment in the code: 
"""
Running <out_path+".py"> file to recreate the original onnx model.
Example usage:
python <out_path+".py"> out_model_path.onnx
"""